### PR TITLE
Trades to offers - More functional

### PR DESCRIPTION
**Pull request by D4nte**
Do some rename as per @da-kami's [comment](https://github.com/coblox/bobtimus/pull/91#discussion_r309989543).

Make the code more functional to make testing easier.

### CODE REVIEW

File: tests/rates/testnetMarketMaker.spec.ts [hunk 1]
```diff
@@ -56,14 +56,14 @@ describe("Test the TestnetMarketMaker module", () => {
...
+    const offers = await marketMaker.prepareOffersToPublishForAsset(
       buyAsset,
       sellAsset
     );
-    expect(trades).toBeDefined();
+    expect(offers).toBeDefined();
 
-    expect(trades.buy).toEqual(trade.buy); // Based on the publish fraction
-    expect(trades.sell).toEqual(trade.sell);
+    expect(offers.buy).toEqual(offer.buy); // Based on the publish fraction
```
Comment (bonomat) [inline tests/rates/testnetMarketMaker.spec.ts]: Had to look twice :D maybe you can rename either `offer` or `offers` if you have touch this again.

File: src/sleep.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
-export default function sleep(ms: number) {
```
Comment (bonomat) [inline src/sleep.ts]: Do we need this sleep function outside of the tests as well? 🤔

File: tests/rates/testnetMarketMaker.spec.ts (hunk 1 above)
Comment (D4nte) [reply tests/rates/testnetMarketMaker.spec.ts]: Yep already done in #100

File: src/sleep.ts (hunk 2 above)
Comment (D4nte) [reply src/sleep.ts]: Yep, to wait for the correct block time (ie, refunds)

**Pull request merged**